src/azure-cli/azure/cli/command_modules/marketplaceordering/generated/_help.py

```diff
@@ -0,0 +1,42 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['marketplaceordering'] = '''
+    type: group
+    short-summary: Manage Marketplace Ordering Agreements
+'''
```

Comment:
This should be gone.

Reply:
Will file an issue to codegen

Reply:
https://github.com/Azure/autorest.az/issues/825